lib/exec.mli

```diff
@@ -86,12 +86,10 @@ val get_opam_archive_url : root: Fpath.t -> string -> (string option, [> Rresult
     resolves the transitive dependencies of [packages]. *)
 val run_opam_package_deps : root: Fpath.t -> string list -> (string list, [> Rresult.R.msg]) result
 
-(** [init_local_opam_switch ~opam_switch ~root ~remotes ()] creates a fresh opam state and switch
-    with compiler [compiler] using [root] as OPAMROOT and adds the [remotes] opam repositories
-    to it. *)
+(** [init_local_opam_switch ~opam_switch ~root ~remotes ()] creates a fresh opam state with a single
+    switch using [root] as OPAMROOT and adds the [remotes] opam repositories to it. *)
```

Comment:
Comment should be `[init_opam_and_remotes ~root ~remotes ()]`.

Reply:
Good catch!